src/window.rs

```diff
@@ -380,6 +395,45 @@ impl Window {
     pub fn set_decorations(&self, decorations: bool) {
         self.window.set_decorations(decorations)
     }
+    
+    /// Centers the window on the current monitor the window lives in.
+    /// 
+    /// ## Platform-specific
+    /// 
+    /// This only works on Windows and X11.
+    /// It's not tested on macOS.
```

Comment:
It's misleading to say that this isn't tested on macOS - it just isn't implemented.